Cargo.toml

```diff
@@ -111,7 +114,7 @@ path = "parity/lib.rs"
 
 [[bin]]
 path = "parity/main.rs"
-name = "parity"
+name = "openethereum"
```

Comment:
we'll need to communicate the binary name change clearly as it might break external services build on top of openethereum